src/components/POLine/const.js

```diff
@@ -1,5 +1,5 @@
-export const ERESOURCE = 'Electronic Resource';
-export const PHYSICAL = 'Physical Resource';
+export const ERESOURCE = 'Electronic resource';
```

Comment:
that might be a dangerous change, since to me it looks like a value we send to the back-end. It's not what we display to user.

Reply:
Yes. You are right. Will check other places

Reply:
fixed